src/index.tsx

```diff
@@ -30,15 +30,16 @@ class WorldMap extends React.Component<Props> {
 }
 
 class UncontrolledWorldMap extends React.Component<{}> {
-  state: State = {selected: null};
+  state: State = {selected: []};
 
   onMapClick = (area) => {
     this.setState(
       function() {
-        if (this.state.selected === area) {
-          return {selected: null};
+        const idx = this.state.selected.indexOf(area);
```

Comment:
I'm nitpicking, but can we use `index` instead of `idx`? I find that avoiding shorthand helps the brain move along smoother when reading.